### PR TITLE
Include Openshift CLI Library

### PR DESCRIPTION
**Pull request by mrrajan**
Signed-off-by: Rajan Ravi <rravi@redhat.com>

This PR includes changes for [robotframework-openshift](https://github.com/red-hat-data-services/robotframework-openshift.git) library.
With this library, user can login to OC CLI using the API URL, username and password. This will help us to configure CLI based interactions via Robotframework.

### CODE REVIEW

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -255,6 +257,11 @@ def get_osd_cluster_info(self):
             yaml_file.write(yaml.dump(data, default_flow_style=False, sort_keys=False))
         log.info("success!")
 
+    def get_osd_cluster_api_url(self, console_url):
```
Comment (psathyan) [inline utils/scripts/ocm/ocm.py:259]: Please add a docString to the method.

File: utils/scripts/ocm/ocm.py [hunk 2]
```diff
@@ -245,6 +245,8 @@ def get_osd_cluster_info(self):
             data = yaml.safe_load(fh)
         console_url = self.get_osd_cluster_console_url()
         data["OCP_CONSOLE_URL"] = console_url
+        cluster_api_url = self.get_osd_cluster_api_url(console_url)
+        data["OCP_API_URL"] = cluster_api_url
```
Comment (psathyan) [inline utils/scripts/ocm/ocm.py]: Let's do the below code as `cluster_api_url` is not referenced anywhere else in the code.
```
data["OCP_API_URL"] = self.get_osd_cluster_api_url(console_url)
```

**Review by harshkumarRH — APPROVED** [review]
Looks good!

**Review by ikanias — APPROVED** [review]
LGTM

**Pull request merged**